roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -1,10 +1,20 @@
 ---
 - block:
-    - copy:
-        src: elasticsearch-logging.yml
+    - fail:
```

Comment:
can we move this fail to the top of `install_logging` instead?
we also want to verify that a valid value was provided